### PR TITLE
schemaPattern parameter - for Oracle JPA generation

### PR DESCRIPTION
**Pull request by karsany**

### CODE REVIEW

File: src/main/java/com/smartnews/jpa_entity_generator/metadata/TableMetadataFetcher.java [hunk 1]
```diff
@@ -31,7 +31,7 @@ private DatabaseMetaData getMetadata(JDBCSettings settings) throws SQLException
         DatabaseMetaData databaseMeta = getMetadata(jdbcSettings);
         try {
             List<String> tableNames = new ArrayList<>();
-            try (ResultSet rs = databaseMeta.getTables(null, null, "%", TABLE_TYPES)) {
+            try (ResultSet rs = databaseMeta.getTables(null,  jdbcSettings.getSchemaName(), "%", TABLE_TYPES)) {
```
Comment (seratch) [inline src/main/java/com/smartnews/jpa_entity_generator/metadata/TableMetadataFetcher.java]:
```
ResultSet getTables​(String catalog, String schemaPattern, String tableNamePattern, String[] types) throws SQLException
```
https://docs.oracle.com/en/java/javase/11/docs/api/java.sql/java/sql/DatabaseMetaData.html#getTables(java.lang.String,java.lang.String,java.lang.String,java.lang.String%5B%5D)

According to the javadoc, it's not a concrete schema name. `JDBCSettings#getSchemaPattern` may be more suitable name for it.

**Review by seratch — COMMENTED** [review]
I left a comment about the naming. 

Apart from that, I believe we can ignore the build failure with Oracle JDK 8.

https://travis-ci.org/smartnews/jpa-entity-generator/jobs/565422342

```
Installing oraclejdk8
$ export JAVA_HOME=~/oraclejdk8
$ export PATH="$JAVA_HOME/bin:$PATH"
$ ~/bin/install-jdk.sh --target "/home/travis/oraclejdk8" --workspace "/home/travis/.cache/install-jdk" --feature "8" --license "BCL"
Ignoring license option: BCL -- using GPLv2+CE by default
install-jdk.sh 2019-07-17
Expected feature release number in range of 9 to 14, but got: 8
The command "~/bin/install-jdk.sh --target "/home/travis/oraclejdk8" --workspace "/home/travis/.cache/install-jdk" --feature "8" --license "BCL"" failed and exited with 3 during .
Your build has been stopped.
```

**Comment by karsany** [comment]
naming changed.

**Comment by seratch** [comment]
Thanks for the contribution!

**Comment by seratch** [comment]
I've shipped a new version (0.99.8) including this change.

**Pull request merged**